src/decompression.jl

```diff
@@ -63,10 +60,25 @@ function compress(A, result::AbstractColoringResult{structure,:row}) where {stru
     return B
 end
 
+function compress(
+    A, result::AbstractColoringResult{structure,:bidirectional}
+) where {structure}
+    row_group = row_groups(result)
+    column_group = column_groups(result)
+    Br = stack(row_group; dims=1) do g
+        dropdims(sum(A[g, :]; dims=1); dims=1)
```

Comment:
Can we avoid the allocation of the slice `A[g, :]` here?

Reply:
Yes, let's do it. Although note that `compress` is mostly for our own use in testing.

Reply:
Actually I tried but this completely changes the nature of the compressed object (which becomes a `SparseMatrixCSC` instead of a `Matrix`, even though it is dense. So I reverted it and I suggest we leave it that way.

Reply:
In particular it made some tests fail.